bundles/org.eclipse.ui.ide/src/org/eclipse/ui/internal/ide/ChooseWorkspaceDialog.java

```diff
@@ -464,6 +471,14 @@ protected Combo createPathCombo(Composite panel) {
 					}
 				}
 				okButton.setEnabled(nonWhitespaceFound);
+				try {
+				File location = new File(characters);
+				label.setText(
+						IDEWorkbenchMessages.ChooseWorkspaceDialog_ResolvedAbsolutePath0 + location.getCanonicalPath());
```

Comment:
We shouldn't show the canonical path here as we most likely don't want to resolve symlinks. What you want is more likely a normalized path. Something like `location.getAbsoluteFile().toPath().normalize()`

Reply:
Thanks for the suggestion. I tried this and it works.